### PR TITLE
Updates on GIF tab

### PR DESCRIPTION
**Pull request by dhruv282**
-Resized GIF to match the size of the slider tab.
-Added download GIF button.
-Added timestamp option. This is an option to have a watermark of the appropriate timestamp on the individual images.
-Got rid of the imagesData array in gif.js because imagesData_IG in imageGridScript.js has the same data and its global so there's no need for imagesData.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -156,9 +156,16 @@ <h3 class="collection_stats_container"><span class="collection_stats"></span> &n
 						<div id="gifContent" style="text-align: center">
 							<div class="intervalTab">		
 								<label class="inputTitle">Animation Interval</label>
-								<input type="number" name="in" min=".1" step=".1" value="0.5" id="interval">
+								<input type="number" name="in" min=".1" step=".1" value="1" id="interval">
 								<div class="inputControl-addon">sec</div>	
+
+								<label id= "watermarkLabel">
+									<input type="checkbox" id="watermarkOption">
+									Add date and time stamps
```
Comment (ibnesayeed) [inline public/index.html]: I think you should change this phrase to something like the following (or better):

```suggestion
									Add Timestamp
```

File: public/styles/sitestyle.css [hunk 2]
```diff
@@ -349,3 +349,8 @@ body{
 #gifApp{
 	margin: 10px 0 0;
 }
+
+.inputControl-addon{
+	margin-right: 30px
```
Comment (ibnesayeed) [inline public/styles/sitestyle.css]: There is a semicolon missing here.

```suggestion
	margin-right: 30px;
```

**Comment by ibnesayeed** [comment]
@dhruv282 did you test it locally and is it ready to be merged and deployed?

**Comment by dhruv282** [comment]
Yes, I did and I'm sure it's ready to be merged and deployed.

**Comment by ibnesayeed** [comment]
Updated code is deployed now.

**Pull request merged**